syntax = "proto3";

package echo;

// A tiny echo service.
service Echo {
  // Unary call: send message, get message back.
  rpc EchoOnce (EchoRequest) returns (EchoReply) {}
}

// Request message.
message EchoRequest {
  string message = 1;
}

// Response message.
message EchoReply {
  string message = 1;
  string received_at = 2;
}
